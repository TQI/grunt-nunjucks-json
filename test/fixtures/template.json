{
	"title": "Olá",
	"names": [
		"Daniel",
		"Flávia"
	],
	"table-1": {

	}
}