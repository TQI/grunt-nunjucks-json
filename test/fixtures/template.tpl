<h1>{{ title }}</h1>
<ul>
	{% for item in names -%}
	{% include "./includes/item.tpl" %}
	{% endfor %}

	{%- set item = "Clara" %}
	{%- include "./includes/item.tpl" %}
</ul>