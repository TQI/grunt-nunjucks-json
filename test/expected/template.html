<h1>Olá</h1>
<ul>
	<li>Daniel</li>
	<li>Flávia</li>
	<li>Clara</li>
</ul>